static/app/views/dashboards/widgetBuilder/components/rpcToggle.tsx

```diff
@@ -0,0 +1,36 @@
+import {Flex} from 'sentry/components/container/flex';
+import SwitchButton from 'sentry/components/switchButton';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {useLocalStorageState} from 'sentry/utils/useLocalStorageState';
+import {useQueryParamState} from 'sentry/views/dashboards/widgetBuilder/hooks/useQueryParamState';
+
+export const DASHBOARD_RPC_TOGGLE_KEY = 'dashboards-spans-use-rpc';
+
+function RPCToggle() {
+  const [_, setIsRpcEnabled] = useQueryParamState<boolean>({
+    fieldName: 'useRpc',
+  });
```

Comment:
is it worth adding this as an option in the QueryParamState as opposed to defining it here?

Reply:
Do you mean the local storage?

side note: tbh I probably could've just used `useNavigate` to control setting the URL params but since this is supposed to be short lived, I don't think it's worth changing if it works for now lol

Reply:
Oh i meant adding it as an option in `useWidgetBuilderState.tsx`. But honestly if it's gonna be short lived it's fine!